### DISCUSSION TITLE
TLS CERT serial number as DEC

### DISCUSSION THREAD

**Discussion by dugaquilin** [nodejs/help#4921] [Ideas]
As per 
https: //nodejs.org/api/tls.html

the CERT serial number is returned as a HEX string, which is wonderful.

However, some software needs the DEC serial number.
I would like to recommend that an API be added that returns the DEC serial number.

The long length HEX string is a pig to convert to a long length DEC string.
Even the Entrust page with CERT details shows the serial number in both HEX and DEC.
Thank you for your consideration

**Comment by bnoordhuis** [comment]
Assuming DEC stands for "decimal": `BigInt("0x" + hex)`

E.g., `BigInt("0xDEADBEEF")` -> `3735928559n`
